### DISCUSSION TITLE
Who is the only person to be starred in every category on the Hollywood Walk of Fame?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#805] [Galaxy Trivia] [answered]
**Bonus:** Pin 📌 the location on the map.

**Comment by SANKARAMDAS** [comment]
Gene Autry is the only honoree with stars in all five categories.

https://blog.batchgeo.com/map-of-the-hollywood-walk-of-fame/     Gene Autry

**Comment by peterbaccaro** [comment, marked as answer]
The only person to have been honored with stars in all five categories on the Hollywood Walk of Fame is the American entertainer and television host, **Gene Autry**. Gene Autry was awarded stars in the following categories:

- Motion Pictures
- Television
- Radio
- Recording
- Live Performance (Theater/Live Performance)

Autry, often referred to as "The Singing Cowboy," had a successful career in various entertainment mediums, including film, television, radio, and music. He became a popular figure in the 1930s and 1940s for his roles in Western films and his hit recordings. Autry's contributions to multiple aspects of the entertainment industry earned him this unique distinction on the Hollywood Walk of Fame.

https://www.google.com/maps/d/edit?mid=1KZzISmRZfqe-e-U7rnzHb-xy4cPPehs&usp=sharing

![image](https://github.com/ParanoidUser/yolo/assets/74666079/4c9d7e93-b4cd-4bfc-ba38-58ba4cf6cdb5)
